File: qasdk/src/main/java/co/astrnt/qasdk/utils/SectionInfo.java
package co.astrnt.qasdk.utils;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class SectionInfo extends RealmObject {

    @PrimaryKey
    private long id;
    private int index;

    public SectionInfo() {
    }

    public SectionInfo(int index) {
        this.index = index;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }

    public void increaseIndex() {
        this.index++;
    }
}

File: qasdk/src/main/java/co/astrnt/qasdk/utils/SectionInfo.kt
package co.astrnt.qasdk.utils

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class SectionInfo : RealmObject {
    @PrimaryKey
    var id: Long = 0
    var index = 0

    constructor() {}
    constructor(index: Int) {
        this.index = index
    }

    fun increaseIndex() {
        index++
    }
}